scarlet/display.py

```diff
@@ -271,7 +282,6 @@ def show_scene(
                 ax[panel].text(*center_[::-1], k, color="w")
 
     fig.tight_layout()
-    plt.close()
```

Comment:
Can you confirm that the notebooks don't show the figures twice without this line?

Reply:
This is why I had to modify the notebooks. The reason that it was showing twice was the this method returns the figure, which is good because it gives users the option of modifying it. So you'll notice in the docs that I had to add a `plt.show()` method, which displays the current figure. Really any line after the `show_scene` function would prevent the figure from displaying twice, but `plt.show()` is the right way to do it.